packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -127,8 +129,147 @@ describe("<DateRangePicker>", () => {
 
             renderDateRangePicker({ initialMonth, maxDate, minDate });
 
-            assert.equal(dateRangePicker.state.displayYear, YEAR);
-            assert.equal(dateRangePicker.state.displayMonth, Months.DECEMBER);
+            assert.equal(dateRangePicker.state.leftView.getYear(), YEAR);
+            assert.equal(dateRangePicker.state.leftView.getMonth(), Months.DECEMBER);
+        });
+    });
+
+    describe("left/right calendar when not sequential", () => {
+        it("only shows one calendar when minDate and maxDate are in the same month", () => {
+            const contiguousCalendarMonths = false;
+            const minDate = new Date(2015, Months.DECEMBER, 1);
+            const maxDate = new Date(2015, Months.DECEMBER, 15);
+
+            renderDateRangePicker({ contiguousCalendarMonths, maxDate, minDate });
+            assert.lengthOf(document.getElementsByClassName("DayPicker"), 1);
+            assert.lengthOf(document.getElementsByClassName(".DayPicker-NavButton--prev"), 0);
+            assert.lengthOf(document.getElementsByClassName(".DayPicker-NavButton--next"), 0);
```

Comment:
We don't want the dots in the class name when using `document.getElementsByClassName`. Won't return anything.